devbuild.sh

```diff
@@ -21,10 +21,10 @@ OPTIONS
       CCPP suites (CCPP_SUITES) to include in build; delimited with ','
   --enable-options="OPTION1,OPTION2,..."
       enable ufs-weather-model options; delimited with ','
-      (e.g. 32BIT | INLINE_POST | UFS_GOCART | MOM6 | CICE6 | WW3 | CMEPS)
+      (e.g. UFS_GOCART | MOM6 | CICE6 | WW3 | CMEPS)
   --disable-options="OPTION1,OPTION2,..."
       disable ufs-weather-model options; delimited with ','
-      (e.g. 32BIT | INLINE_POST | UFS_GOCART | MOM6 | CICE6 | WW3 | CMEPS)
+      (e.g. UFS_GOCART | MOM6 | CICE6 | WW3 | CMEPS)
```

Comment:
We likely don't want to remove these options. Is this feature broken for more recent versions of the model?

Reply:
@christinaholtNOAA, Have you tried these options? These two don't work with the recent version of the ufs weather model for me. Currently, we set these two options in `sorc/CMakeLists.txt`.

Reply:
change removed.